src/components/portal/portal.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { createPortal } from 'react-dom';
 import Browser from '../../utils/helpers/browser';
-
+import ReactDOM from 'react-dom';
```

Comment:
this probably needs moving up in order, I'd guess eslint will complain about it. Also best keep the line break between the imports and the class.